shell/plugins/dashboard-store/actions.js

```diff
@@ -208,7 +213,9 @@ export default {
         commit('forgetType', type);
       }
 
-      dispatch('loadDataPage', { type, opt: pageFetchOpts });
+      dispatch('loadDataPage', {
+        type, opt: pageFetchOpts, namespace: opt.namespaced
```

Comment:
do we need to break down `namespace` to another arg since it's included on `pageFetchOpts` by default?

Reply:
updated